### PR TITLE
Leaderboard: setup project

### PR DESCRIPTION
**Pull request by Danfall369**
Hello, Code Reviewer

I set up this branch with the following tools and files
- Linters
- Webpack
- Html/Css/js files to src folder and build into to the dist folder

I have completed the requirement to create an npm project with webpack and write plain HTML markup using ES6 modules with import and export. I have used the wireframe provided as a guide to creating the basic UI.

Please review my pull request and let me know if there are any further changes needed. Thank you for your time and consideration.

Best regards

### CODE REVIEW

**Comment by huseyinbiyik** [comment]
Hi 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Alt Text

## Invalid Code Review Request

⚠️  Kindly use [Microverse Readme template](https://github.com/microverseinc/readme-template) and modify the file to match your project and remove sections that don't apply. Please remove all the instructions targeting you. For example: "Describe how a new developer could make use of your project." Please remove the template logo on the top, if you have your own logo you can replace it with this one, otherwise please delete it. This is crucial to have a professional and instructive repository. 👌 
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,2 +1,174 @@
...
+Install this project with:
+
+N/A
+
+### Usage
+
+To run the project, execute the following command:
+
+<p>-With Visual Studio Code, open the folder where you downloaded this repository</p>
+
```
Comment (Gambit142) [inline README.md:103]: - [OPTIONAL] To install this project, you need to run `npm install`. The `npm install` command installs all the packages in the package.json file used to build, start and run the application. 👍 💻 

To use this application, you need to run `npm start`. The `npm start` will start the application on the development server. 👍

File: package.json [hunk 2]
```diff
@@ -0,0 +1,648 @@
...
+    "xdg-basedir": "^4.0.0",
+    "xml-name-validator": "^4.0.0",
+    "xmlchars": "^2.2.0",
+    "xregexp": "^2.0.0",
+    "yallist": "^4.0.0",
+    "yaml": "^1.10.2",
+    "yargs-parser": "^20.2.9",
+    "yauzl": "^2.10.0",
+    "zwitch": "^1.0.5"
+  }
```
Comment (Gambit142) [inline package.json:647]: - [OPTIONAL] Most of the packages you specified in the dependencies section in the package.json file are not needed for this application to run. I would advise you to keep it simple and use the packages that are needed. 😄

**Review by Gambit142 — APPROVED** [review]
Hi @Danfall369 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- All linters checks passed ✔️ 
- Use of `webpack` ✔️ 
- Design looks identical to the wireframe provided ✔️ 

### Optional Suggestions
- Kindly consider all the comments tagged [OPTIONAL] below the review summary. 👍 😎 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Danfall369) [reply README.md:103]: @Gambit142 Thanks, I'll do it right now

File: package.json (hunk 2 above)
Comment (Danfall369) [reply package.json:647]: @Gambit142 All those files appear when I make an npm init -y

**Pull request merged**